lib/Header.js

```diff
@@ -1,37 +1,93 @@
-import React from "react";
-import PropTypes from "prop-types";
-import { Text, utils, widgetCreator } from "@bpanel/bpanel-ui";
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Text, utils, widgetCreator } from '@bpanel/bpanel-ui';
 
 const { connectTheme } = utils;
 
-const Header = ({ statusIcon, network, bcoinUri, theme }) => (
-  <div className="container">
-    <div
-      className={`${theme.headerbar.networkStatus} ml-md-auto text-right col`}
-    >
-      <div className="network text-uppercase">
-        <Text className={theme.headerbar.text}>
-          Status: {network}
-          <i
-            className={`${theme.headerbar.icon} fa fa-${statusIcon}`}
-            areahidden="true"
-          />
-        </Text>
-      </div>
-      <div className="node">
-        <Text className={`${theme.headerbar.nodeText} ${theme.headerbar.text}`}>
-          Node:{" "}
-        </Text>
-        <Text className={`${theme.headerbar.text}`}>{bcoinUri}</Text>
+const Clients = ({theme, clients, onConfigChange}) => {
+  let clientIds = (
+      <Text className={`${theme.headerbar.text}`}>
+        {clients.currentClient.id}
+      </Text>
+    );
+
+  const ids = Object.keys(clients.clients);
+  if (ids.length)
+    clientIds = (
+      <select
```

Comment:
No `bpanel-ui/Dropdown`?

Reply:
I considered it but I thought it was simpler in the UI to just use basic `select`.